README.md

```diff
@@ -1 +1,46 @@
-# tech-academy
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Tech Academy
+
+An online website for a Tech academy that show the available courses and the top instructors   
+
+
+## Built With
+
+- HTML ,CSS, JavaScript
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+git clone git@github.com:Zainab-Alzubair/tech-academy.git
+
+## Online version
+https://zainab-alzubair.github.io/tech-academy/
```

Comment:
- [ ] Its great that you have added your deployment link in the readme file :blush: it would be nice if you add your loom video link here as well. I can see that you have included that in your pull request summary. Kindly add it in your readme as well :+1: